src/index.js

```diff
@@ -30,7 +30,7 @@ const reduceMatchedKeyPaths = (obj, keyPath) => {
 }
 
 const loader = function (content) {
-  if (typeof this.query === 'string') {
+  if (typeof this.query !== 'string') {
```

Comment:
I think you may misunderstand the meaning of `this.query`, here is the example for `this.query` https://github.com/foray1010/Popup-my-Bookmarks/blob/7103158b007753b83dd29fddedff73832bf99415/webpack.config.js#L140-L142